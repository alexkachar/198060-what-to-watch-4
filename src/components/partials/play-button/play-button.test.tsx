import * as React from 'react';
import * as renderer from 'react-test-renderer';
import PlayButton from './play-button';

it(`PlayButton renders correctly`, () => {
  const tree = renderer
    .create(
        <PlayButton
          isPlaying={true}
          onPlayButtonClick={jest.fn()}
        />, {
          createNodeMock: () => document.createElement(`div`)
        })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
