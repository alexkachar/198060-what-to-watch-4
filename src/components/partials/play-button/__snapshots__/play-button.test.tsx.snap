// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayButton renders correctly 1`] = `
<button
  className="player__play"
  onClick={[MockFunction]}
  type="button"
>
  <svg
    height={19}
    viewBox="0 0 19 19"
    width={19}
  >
    <use
      xlinkHref="#pause"
    />
  </svg>
  <span>
    Play
  </span>
</button>
`;
